intermediate/remote_data_access.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    zos_2015jan = ds.zos.sel(time="2015-01-16").isel(time=0)
</pre><p>Why the double selection? Shouldn't the first <code>.sel()</code> do the trick? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Yeah this is a weirdness around `.sel` with datetimes. I would stick `.squeeze()` at the end maybe. Or replace with `.sel(time="2015-01-16 00:00")`

Reply:
Decided to go with `.squeeze()`, thanks for the tip @dcherian! Done in e96181b89f369e1358ee421de0529cfee5429a48.